internal/scorecard/alpha/tests/test_bundle.go

```diff
@@ -16,25 +16,21 @@ package tests
 
 import (
 	"bytes"
-	"encoding/json"
 	"fmt"
+	"os"
 
 	"github.com/operator-framework/api/pkg/manifests"
-	"github.com/operator-framework/api/pkg/validation/errors"
 	"github.com/operator-framework/operator-registry/pkg/registry"
 	"github.com/sirupsen/logrus"
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-// TestBundle holds the bundle contents to be tested
-type TestBundle struct {
-	BundleErrors []errors.ManifestResult
-	Bundles      []*registry.Bundle
-	CRs          []unstructured.Unstructured
-}
+// GetBundle parses a Bundle from a given on-disk path returning a bundle
+func GetBundle(bundlePath string) (bundle registry.Bundle, err error) {
```

Comment:
Since there's a possibility of returning an error, it is generally idiomatic to either return `nil, error` or `bundle, nil`, but never both `bundle, err`, unless the returned item is partially usable even with the error.
```suggestion
func GetBundle(bundlePath string) (bundle *registry.Bundle, err error) {
```

Reply:
ok, fixed this just now.